TestCases/parallel_regression_AD.py

```diff
@@ -328,7 +328,7 @@ def main():
     da_sp_pinArray_cht_2d_dp_hf.cfg_dir   = "incomp_navierstokes/streamwise_periodic/chtPinArray_2d"
     da_sp_pinArray_cht_2d_dp_hf.cfg_file  = "DA_configMaster.cfg"
     da_sp_pinArray_cht_2d_dp_hf.test_iter = 100
-    da_sp_pinArray_cht_2d_dp_hf.test_vals = [-4.793283, -4.065832, -4.137121] #last 4 lines
+    da_sp_pinArray_cht_2d_dp_hf.test_vals = [-4.800583, -4.065533, -4.137185] #last 4 lines
```

Comment:
Well the deviations are really tiny but both inc cases have `INC_ENERGY_EQUATION= YES` but `discadj_cht` is not here as the third case that has that... Not that I am worried but if you have an explanation why exactly those cases differ, I'd be interested?

Reply:
The common denominator seemed to be cases that had a significant number of linear iterations.
I ran them to convergence and there is 0 difference in final results.
I will still test these changes in my optimization cases before merging.